.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -219,4 +235,29 @@ stages:
           inputs:
             artifactName: 'publishing-info'
             targetPath: 'vhd-publishing-info.json'
-          condition: and(succeeded(), eq(variables.DRY_RUN, 'False'), ne(variables['MODE'], 'sigMode'))
\ No newline at end of file
+          condition: and(succeeded(), eq(variables.DRY_RUN, 'False'), ne(variables['MODE'], 'sigMode'))
+  - stage: build_arm64_vhd
+    dependsOn: []
+    condition: and(eq('${{ parameters.buildarm64vhd }}', true), eq(variables.SIG_GALLERY_NAME, ''), eq(variables.SIG_IMAGE_NAME, ''), eq(variables.SIG_IMAGE_VERSION, ''))
+    jobs:
+    - job: build
+      timeoutInMinutes: 180
+      steps:
+        - bash: |
+            echo '##vso[task.setvariable variable=DRY_RUN]True'
+            echo '##vso[task.setvariable variable=OS_SKU]Ubuntu'
+            echo '##vso[task.setvariable variable=OS_VERSION]18.04'
+            echo '##vso[task.setvariable variable=IMG_PUBLISHER]Canonical'
+            echo '##vso[task.setvariable variable=IMG_OFFER]0002-com-ubuntu-server-arm-preview-bionic'
+            echo '##vso[task.setvariable variable=IMG_SKU]18_04-lts'
+            echo '##vso[task.setvariable variable=HYPERV_GENERATION]V2'
+            echo '##vso[task.setvariable variable=AZURE_LOCATION]westus2'
+            echo '##vso[task.setvariable variable=AZURE_VM_SIZE]Standard_D8pds_V5'
```

Comment:
use smaller sku, e.g. Standard_D2pds_v5 to run arm64 check-in test, arm64 azure resources are still very limited.